commands/user.go

```diff
@@ -104,30 +114,47 @@ func init() {
 	RootCmd.AddCommand(UserCmd)
 }
 
-func deactivateUsers(c client.Client, userArgs []string) {
+func userActivateCmdF(c client.Client, command *cobra.Command, args []string) error {
+	if len(args) < 1 {
+		return errors.New("Expected at least one argument. See help text for details.")
+	}
+
+	changeUsersActiveStatus(c, args, true)
+
+	return nil
+}
+
+func changeUsersActiveStatus(c client.Client, userArgs []string, active bool) {
 	users := getUsersFromUserArgs(c, userArgs)
 	for i, user := range users {
-		if user == nil {
-			printer.PrintError("Unable to find user '" + userArgs[i] + "'")
-			continue
-		}
+		err := changeUserActiveStatus(c, user, userArgs[i], active)
 
-		if user.IsSSOUser() {
-			printer.Print("You must also deactivate user " + userArgs[i] + " in the SSO provider or they will be reactivated on next login or sync.")
+		if err != nil {
+			printer.PrintError(err.Error())
 		}
+	}
+}
 
-		if _, response := c.DeleteUser(user.Id); response.Error != nil {
-			printer.PrintError("Unable to deactivate user " + userArgs[i] + ". Error: " + response.Error.Error())
-		}
+func changeUserActiveStatus(c client.Client, user *model.User, userArg string, activate bool) error {
```

Comment:
I don't think we are using `c.DeleteUser` anywhere else. If so, would you mind removing it from the `Client` and updating the mocks?